### PR TITLE
KOGITO-5533: add examples of evaluation of rules via CloudEvents

### PR DESCRIPTION
**Pull request by kostola**
* [Jira task](https://issues.redhat.com/browse/KOGITO-5533)

PRs
- https://github.com/kiegroup/kogito-runtimes/pull/1610
- https://github.com/kiegroup/kogito-examples/pull/886

### CODE REVIEW

**Comment by danielezonca** [comment]
Let's wait this other PR to be merged https://github.com/kiegroup/kogito-runtimes/pull/1554

After that PR also this addon will become a quarkus extension so it will require some work

@ricardozanini FYI

File: ruleunit-event-driven-quarkus/src/test/java/org/kie/kogito/examples/NativeRuleUnitEventDrivenIT.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * limitations under the License.
+ */
+package org.kie.kogito.examples;
+
+import org.kie.kogito.testcontainers.quarkus.KafkaQuarkusTestResource;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.NativeImageTest;
+
+@NativeImageTest
```
Comment (danielezonca) [inline ruleunit-event-driven-quarkus/src/test/java/org/kie/kogito/examples/NativeRuleUnitEventDrivenIT.java]: Can you use `@QuarkusIntegrationTest`?

File: ruleunit-event-driven-quarkus/src/test/resources/events/query/find_approved/input.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (danielezonca) [inline ruleunit-event-driven-quarkus/src/test/resources/events/query/find_approved/input.json:1]: Please add a test also with a wrong input, wrong query, etc similar to DMN example

Comment (kostola) [reply ruleunit-event-driven-quarkus/src/test/resources/events/query/find_approved/input.json:1]: Done :+1:

File: ruleunit-event-driven-quarkus/src/test/java/org/kie/kogito/examples/NativeRuleUnitEventDrivenIT.java (hunk 1 above)
Comment (kostola) [reply ruleunit-event-driven-quarkus/src/test/java/org/kie/kogito/examples/NativeRuleUnitEventDrivenIT.java]: Done :+1:

**Review by jiripetrlik — APPROVED** [review]
LGTM

**Review by danielezonca — APPROVED** [review]

**Pull request merged**